generator/resources/ec2_performance_test_matrix.json

```diff
@@ -6,7 +6,7 @@
         "ami": "cloudwatch-agent-integration-test-al2*",
         "arc": "amd64",
         "binaryName": "amazon-cloudwatch-agent.rpm",
-        "valuesPerMinute": 10,
+        "valuesPerMinute": 100,
```

Comment:
Changing the metrics/logs/traces sending per minutes to reflect more performance metrics instead of 10/100/1000.